app/src/main/java/com/github/ybecker/epforuml/sensor/CameraActivity.kt

```diff
@@ -116,6 +159,84 @@ class CameraActivity : AppCompatActivity() {
 
     }
 
+    private fun startRecordingVideo() {

```

Comment:
This function might be a bit hard to read. It would have been nice to separate it a bit more